src/compiler/crystal/semantic/call_error.cr

```diff
@@ -71,6 +71,12 @@ class Crystal::Call
       end
     end
 
+    # Also check with scope
+    with_scope = @with_scope
```

Comment:
I don't inline it because it's also used later on.

Reply:
```cr
if foo = @foo 
  # ...
end
# foo is still usable here
```
But I understand. To make it more clear, I'd add a blank line after the assign:
```
foo = @foo 

if foo
```
To show that it's not only for the `if`

Reply:
Good idea! Done, and also moved another var like that.